.babelrc

```diff
@@ -1,9 +1,8 @@
 {
 	"presets": [
-		[ "latest", {
-			"es2015": {
-				"modules": false
-			}
-		} ]
+		[ "latest", "es2015" ]
```

Comment:
Is there a further change we should make here?  I'm not sure if you can do `latest: { modules: false }` for example, or how to test whether this is working as intended.